package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "devsign-server",
+  "version": "1.0.0",
+  "description": "Express/MongoDB for Devsign/Twitter-clone: Chirp",
+  "main": "server.js",
+  "dependencies": {
+    "auth0": "^2.15.0",
+    "chance": "^1.0.18",
+    "dotenv": "^6.2.0",
+    "express": "^4.16.4",
+    "express-jwt": "^5.3.1",
+    "jwks-rsa": "^1.4.0",
+    "mongoose": "^5.4.6",
+    "superagent": "^4.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^5.12.1",
+    "jest": "^23.6.0",
+    "morgan": "^1.9.1",
+    "nodemon": "^1.18.9",
+    "supertest": "^3.4.2"
+  },
+  "scripts": {
+    "lint": "eslint .",
```

```suggestion
    "lint": "eslint .",
    "pretest: "npm run lint",
```
Don't forget the pretest script